### ISSUE TITLE
Make a decision about editorial owner and attributed author controls

### ISSUE THREAD

**Issue by johnbillion** [humanmade/authorship#66]
Removing the built-in "Author" control (which maps directly to the `post_author` field) may not be as desirable as we originally thought because there can still be a need for Editor and Administrator level users to change this value.

Authorship provides a means of attributing multiple, different users to a post. The post author (from the `post_author` field) may or may not be in the list of attributed users. With Authorship active this field indicates the editorial owner of the post rather than the attributed author.

On the SW project there is no need for a user to be able to control this value, but on ING there could be.

Note that this change only affects Editors and Administrators.

## Options

1. Reinstate the built-in Author control
   - Allows control over the editorial owner
   - Complicates the UI, two different controls for selecting users
2. Introduce a combo UI for selecting attributed authors and the editorial owner in one control
   - As suggested during one of our review calls
   - New UI, could be better but will be less standard
   - Avoids the need for two different controls for selecting users
3. Replace the built-in Author control with another instance of the `AuthorSelect` component for selecting only one user
   - More consistent UI
4. Keep what we have now which is just one piece of UI for the attributed authors
   - Simpler UI overall
   - Doesn't allow controlling the editorial user

## Related

* #1 
* #15

**Comment by roborourke** [comment]
Do you have a particular favourite of these options @johnbillion ?

**Comment by johnbillion** [comment]
I really like the idea of 2 but it means introducing the concept of editorial owners as well as attributed users. I can see this being really useful for certain projects but it might not be necessary for most.

**Comment by johnbillion** [comment]
Perhaps option 2 as an opt-in feature?

**Comment by roborourke** [comment]
Yeah, being able to control editorial owner is really important I think, especially with regards to the guest author concept.

I would make it opt-out as right now you have no control of the actual post author currently. So maybe a filter to switch it off and use the first author from the list as post author - accounting for permissions etc...
